### PR TITLE
update calico-policies

### PR DESCRIPTION
**Pull request by sanchezma5**
- remove decommissioned IPs
- add policies for eu-es

### CODE REVIEW

**Review by bradbehle — CHANGES_REQUESTED** [review]

File: calico-policies/private-network-isolation/au-syd/allow-egress-pods-private.yaml [hunk 1]
```diff
@@ -92,9 +91,6 @@ spec:
       - 166.9.56.33/32
       - 166.9.56.35/32
       - 166.9.56.36/32
-      - 166.9.244.107/32
```
[the remark this answers is not in the document]
Comment (sanchezma5) [reply calico-policies/private-network-isolation/au-syd/allow-egress-pods-private.yaml]: proxy CSE addresses have been restored

**Review by bradbehle — APPROVED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      ports:
+      - 2040:2041
+      nets:
+      - 172.20.0.0/24
+    protocol: TCP
+  - action: Allow
+    destination:
+      selector: projectcalico.org/orchestrator == 'k8s'
+    source: {}
+  # Denies pod egress to private networks. If workers are connected to a public VLAN,
```
Comment (ctrath) [inline calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml:57]: We should probably change this comment.  We are allowing pod egress to private networks here, not denying

**Review by ctrath — APPROVED** [review]

File: calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml (hunk 2 above)
Comment (bradbehle) [reply calico-policies/private-network-isolation/eu-es/allow-egress-pods-private.yaml:57]: Agreed, let's fix this in a separate PR

**Comment by bradbehle** [comment]
Since this is time sensitive, I'm going to override our rule of requiring 2 approvals from @kersten1 @derekpoindexter and myself (code owners for these files) since I don't see them online, and also since @ctrath approved as well, and merge this.

**Pull request merged**